docs/tutorials/sentiment-analysis-of-movie-reviews.mdx

```diff
@@ -0,0 +1,161 @@
+---
+title: Sentiment Analysis of Movie Reviews using OpenAI and MongoDB
+sidebarTitle: Sentiment Analysis of Movie Reviews
+---
+
+## Introduction
+
+In this blog post, we present how to create OpenAI models within MongoDB. This example is a sentiment analysis where we infer emotions behind a text i.e, Movie Reviews.
+
+- we'll use datasets from [`Hugging Face`](https://huggingface.co/datasets/rotten_tomatoes)
+
+## Prerequisites
+
+To follow along, you can sign up for an account at [cloud.mindsdb.com](https://cloud.mindsdb.com/register/nlp). Alternatively, head to [MindsDB documentation](https://docs.mindsdb.com/) and follow the instructions to manually set up a local instance of MindsDB via [Docker](/setup/self-hosted/docker) or [pip](/setup/self-hosted/pip/source).
+
+## How to Connect MindsDB to a Database
+
+We use a collection from our MongoDB public movie reviews database, so let’s start by connecting MindsDB to it.
+
+You can use [Mongo Compass](/connect/mongo-compass) or [Mongo Shell](/connect/mongo-shell) to connect your database, which we downloaded from [`Hugging Face`](https://huggingface.co/datasets/rotten_tomatoes), like this:
+
+## Connecting a Database
+
+We start by connecting a demo database using the CREATE DATABASE statement:
+
+- `host` : connection string for the mongodb
+
+```bash
```

Comment:
Please note that it is a tutorial using MongoDB. Thus, you should use MQL (and not SQL).

Please see an example here: https://docs.mindsdb.com/nlp/sentiment-analysis-inside-mongodb-with-openai

Reply:
@martyna-mindsdb , Sorry for using SQL, I've committed with MQL.